example/lib/app/app.dart

```diff
@@ -114,7 +114,7 @@ class _ChewieDemoState extends State<ChewieDemo> {
       videoPlayerController: _videoPlayerController1,
       autoPlay: true,
       looping: true,
-      progressIndicatorDelayMS: bufferDelay,
+      progressIndicatorDelayMs: bufferDelay,
```

Comment:
Is `bufferDelay` being assigned anywhere? Looks like it's only being declared and not actually used.

Reply:
Yes, it's set at line 321. As we need to restart the player to apply those changes, I restart the player only when we click on 'save'

Reply:
make sense, added